app/models/role.rb

```diff
@@ -4,10 +4,10 @@ class Role < ActiveHash::Base
   include ActiveHash::Enum
 
   self.data = [
-    { id: 0, name: "viewer" },
-    { id: 1, name: "deployer" },
-    { id: 2, name: "admin" },
-    { id: 3, name: "super_admin" }
+    { id: 0, name: "viewer", display_name: "Viewer" },
+    { id: 1, name: "deployer", display_name: "Deployer" },
+    { id: 2, name: "admin", display_name: "Admin" },
+    { id: 3, name: "super_admin", display_name: "Super Admin" }
```

Comment:
Could probably have left the display_name out, and used 'role.name.humanize' when displaying it. But not too concerned.